### PR TITLE
[HR] Documentation cleanup

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

File: lib/ui/painting.dart [hunk 1]
```diff
@@ -1570,13 +1570,20 @@ class FrameInfo extends NativeFieldWrapperClass2 {
 }
 
 /// A handle to an image codec.
+///
+/// This class is created by the engine, and should not be instantiated
```
Comment (amirh) [inline lib/ui/painting.dart]: nit: maybe worth making this a macro

File: lib/ui/isolate_name_server.dart [hunk 2]
```diff
@@ -4,31 +4,67 @@
 
 part of dart.ui;
 
-/// Contains methods to allow for simple sharing of SendPorts across isolates.
-abstract class IsolateNameServer {
-  /// Looks up the [SendPort] associated with a given name. Returns null
-  /// if the name does not exist.
+/// Static methods to allow for simple sharing of [SendPort]s across [Isolates].
```
Comment (bkonyi) [inline lib/ui/isolate_name_server.dart]: Should be `[Isolate]s` for consistency.

File: lib/ui/plugins.dart [hunk 3]
```diff
@@ -4,31 +4,42 @@
...
+  final int _handle;
+
+  /// Get the raw callback handle to pass over a [MethodChannel] or [SendPort]
+  /// (to pass to another [Isolate]).
   int toRawHandle() => _handle;
 
   @override
-  int get hashCode => _handle;
+  bool operator ==(dynamic other) {
+    if (runtimeType != other.runtimeType)
```
Comment (bkonyi) [inline lib/ui/plugins.dart]: Any particular reason for this change here? Does this not accomplish the same thing?

**Comment by chinmaygarde** [comment]
Can you land this?

**Pull request merged**